shardingsphere-scaling/shardingsphere-scaling-bootstrap/pom.xml

```diff
@@ -73,7 +79,13 @@
             <artifactId>postgresql</artifactId>
             <scope>runtime</scope>
         </dependency>
-        
+
+        <dependency>
+            <groupId>org.opengauss</groupId>
+            <artifactId>opengauss-jdbc</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+
```

Comment:
It might be not required, since `shardingsphere-scaling-opengauss` is a dependency.